### PR TITLE
Opening times tests

### PR DESCRIPTION
**Pull request by gestchild**
Relates to #7547

Adding some more tests

### CODE REVIEW

File: common/services/prismic/opening-times.ts [hunk 1]
```diff
@@ -355,14 +355,13 @@ export function parseCollectionVenues(
 export function getTodaysVenueHours(
   venue: Venue
 ): ExceptionalOpeningHoursDay | OpeningHoursDay | undefined {
-  const todaysDate = london().startOf('day');
+  const todaysDate = london();
```
Comment (davidpmccormick) [inline common/services/prismic/opening-times.ts:358]: is `.startOf('day')` superfluous?

File: common/test/services/prismic/opening-times.test.ts [hunk 2]
```diff
@@ -265,10 +274,282 @@ describe('opening-times', () => {
...
+        overrideDate: london('2021-12-21'),
+        overrideType: 'Bank holiday',
+        opens: '10:00',
+        closes: '18:00',
+        isClosed: false,
+      });
+    });
+  });
+
+  describe("backfillExceptionalVenueDays: returns the venue's exceptional opening times for each date, and if there is no exceptional opening time for a specific date, then it uses the venues regular opening times for that day.", () => {
```
Comment (davidpmccormick) [inline common/test/services/prismic/opening-times.test.ts:408]: nit: "venue's" possessive

Comment (davidpmccormick) [inline common/test/services/prismic/opening-times.test.ts:408]: nit: "venue's" possessive

**Review by davidpmccormick — APPROVED** [review]

File: common/services/prismic/opening-times.ts (hunk 1 above)
Comment (gestchild) [reply common/services/prismic/opening-times.ts:358]: No, I just [moved it ](https://github.com/wellcomecollection/wellcomecollection.org/blob/214734da23f9f287657cda72a613d7ed0fa8dd7c/common/services/prismic/opening-times.ts#L363)so it was clearer what was being compared in the find method

**Pull request merged**